frontend/src/app/pages/summary/transactions/transactions.component.ts

```diff
@@ -32,4 +22,17 @@ export class TransactionsComponent implements OnInit {
         );
   }
 
+  private getBankColor(transaction: Transaction): string {
+    switch (transaction.bankId) {
```

Comment:
Please create a global object for that. It won't take long for a code like that to bite you in the ass in the future.

Reply:
Must